docs/DeveloperGuide.md

```diff
@@ -288,24 +295,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | Teaching Assistant who is teaching in multiple semesters  | archive the existing data                                                              | save a copy on the local machine before undergoing a complete deletion of data                |
-| `*`      | Teaching Assistant                                        | tag students                                                                           | know what modules they are taking                                                             |
-| `*`      | Teaching Assistant                                        | delete multiple students from TAPA                                                     | remove students that are no longer being taught, without needing to delete them individually  |
-| `*`      | Teaching Assistant                                        | display student progress                                                               | locate a person easily                                                                        |
-| `*`      | Teaching Assistant                                        | filter students by their tags                                                          | view the progress of students in each separate tag (different class/module)                   |
-| `*`      | Teaching Assistant                                        | update the student’s tags                                                              | edit when the student drop/take a new module                                                  |
-| `*`      | Teaching Assistant                                        | define my own tasks for each module                                                    | customise the tasks to each module                                                            |
-| `*`      | Teaching Assistant                                        | assign tasks to multiple students                                                      | allocate tasks to students without having to individually assign them                         |
+| `* *`    | Teaching Assistant                                        | delete multiple students from TAPA                                                     | remove students that are no longer being taught, without needing to delete them individually  |
+| `* *`    | Teaching Assistant                                        | delete multiple students from TAPA after the module is over                            | remove students that are no longer being taught, after the semester is over                   |
+| `* *`    | Teaching Assistant                                        | display student progress                                                               | locate a person easily                                                                        |
```

Comment:
The third part feels like it doesnt really match the user story (since the user story is focusing on progress instead)

Reply:
Yup, my mistake on this part, did not notice it on the first go. Edited it as well.